src/store/guild_action/remove.rs

```diff
@@ -0,0 +1,54 @@
+use std::sync::Arc;
+
+use anyhow::Result;
+use serenity::async_trait;
+use typed_builder::TypedBuilder;
+
+use super::GuildStore;
+use crate::{
+    command_context::CommandContext,
+    store::guild_action::{Execute, HasCtx},
+};
+
+#[async_trait]
+impl Execute for Remove {
+    async fn execute(&self, store: &mut GuildStore) -> Result<()> {
+        let Remove { ctx, from, to } = self;
+
+        if let Some(to) = to {
+            if from > to {
+                ctx.send(format!(
+                    "To cannot be greater than from, (from: {from}, to: {to})"
```

Comment:
Kan nog köra should not istället, då det är en uppmaning, men de kan fortfarande försöka :p

Reply:
Fixat!